libindi/libs/indibase/basedevice.h

```diff
@@ -20,6 +20,7 @@
 
 #include "indibase.h"
 #include "indiproperty.h"
+#include "dsp.h"
```

Comment:
This shouldn't be in the header as it breaks all 3rd party drivers not using DSP.